src/pages/en/core-concepts/framework-components.md

```diff
@@ -44,11 +44,11 @@ export default defineConfig({
 
 Use your JavaScript framework components in your Astro pages, layouts and components just like Astro components! All your components can live together in `/src/components`, or can be organized in any way you like.
 
-To use a framework component, import it from its relative path (including file extension) in your Astro component script. Then, use the component alongside other components, HTML elements and JSX-like expressions in the component template.
+To use a framework component, import it from its relative path in your Astro component script. Then, use the component alongside other components, HTML elements and JSX-like expressions in the component template.
 
 ```astro
 ---
-import MyReactComponent from '../components/MyReactComponent.jsx';
+import MyReactComponent from '../components/MyReactComponent';
```

Comment:
```suggestion
import MyReactComponent from '../components/MyReactComponent';
import MySvelteComponent from '../components/MySvelteComponent.svelte';
```

(and update the templating)

Do we want to do this, right off the bat, so at least it's OBVIOUS that sometimes there's a file extension, sometimes there isn't? 

I would really want some kind of call out below that explicitly says when an extension is/isn't required. But, I worry about a single example of React, right off the bat, when it's the ONLY one that does NOT require the file extension. It will be too easy for people to get the wrong pattern at a glance?

Reply:
Good shout